hack/e2e.go

```diff
@@ -579,6 +588,13 @@ func KubemarkTest() error {
 	if err != nil {
 		return err
 	}
+	// If we tried to bring the Kubemark cluster up, make a courtesy
+	// attempt to bring it down so we're not leaving resources around.
+  //
+  // TODO: We should try calling stop-kubemark exactly once. Though to
+  // stop the leaking resources for now, we want to be on the safe side
+	// and call it explictly in defer if the other one is not called.
+	defer finishRunning("Stop kubemark", exec.Command("./test/kubemark/stop-kubemark.sh"))
```

Comment:
Can we please put the start/stop kubemark calls inside a xmlWrap("Deferred Stop kubemark", finishRunning(...))? Alternatively clarify why this is a bad idea?

It seems like a useful idea to me because these calls a) take a long time and b) have a modest probability of failing. This it will be really nice if we show that these steps are failing without having to dig through logs (in gubernator, testgrid, jenkins, etc)